samples/modules/lvgl/touch_draw/prj.conf

```diff
@@ -0,0 +1,16 @@
+CONFIG_LV_Z_MEM_POOL_SIZE=32768
+CONFIG_LV_Z_SHELL=y
+CONFIG_MAIN_STACK_SIZE=2048
+
+CONFIG_DISPLAY=y
+CONFIG_DISPLAY_LOG_LEVEL_ERR=y
+
+CONFIG_LOG=y
+CONFIG_SHELL=y
+
+CONFIG_LVGL=y
+CONFIG_LV_MEM_CUSTOM=y
```

Comment:
This is not needed `LV_Z_MEM_POOL_SYS_HEAP` is the default and this symbol is always selected to `y` via `lv_confg.h`